specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2021-12-01-preview/definitions.json

```diff
@@ -4010,6 +4010,180 @@
       },
       "description": "A collection of resources."
     },
+    "PortalConfigCollection": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PortalConfigContract"
+          },
+          "description": "The developer portal configurations."
+        },
+        "count": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Total record count number across all pages."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Next page link if any.",
+          "readOnly": true
+        }
+      },
+      "description": "The collection of the developer portal configurations."
+    },
+    "PortalConfigContract": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PortalConfigProperties",
+          "description": "The developer portal configuration contract properties."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./../../../../../common-types/resource-management/v2/types.json#/definitions/Resource"
+        }
+      ],
+      "description": "The developer portal configuration contract."
+    },
+    "PortalConfigProperties": {
+      "type": "object",
+      "properties": {
+        "enableBasicAuth": {
```

Comment:
nit: for operational Boolean values, recommend add a default value. "default": true | false,

Reply:
Yep, added the defaults.